docs/Tasks/installing-an-extension.md

```diff
@@ -1,62 +1,116 @@
-
-```bash 
-$ kubectl get namespaces | grep argocd 
-argocd-operator-system       Active   4m17s
-
-$ kubectl get pods -n argocd-operator-system 
-NAME                                                 READY   STATUS    RESTARTS   AGE
-argocd-operator-controller-manager-bb496c545-ljbbr   2/2     Running   0          4m32s
-```
+# Installing an extension from a catalog
+
+In Operator Lifecycle Manager (OLM) 1.0, Kubernetes extensions are scoped to the cluster.
+After you add a catalog to your cluster, you can install an extension by creating a custom resource (CR) and applying it.
+
+!!! important
+
+    Currently, extensions that use webhooks or target a single or specified set of namespaces cannot be installed.
+    Extensions must not include webhooks and must use the `AllNamespaces` install mode.
+
+
+## Prerequisites
+
+* The `jq` CLI tool is installed.
+* You have added a catalog to your cluster.
+
+## Procedure
+
+1. Create a CR for the Kubernetes extension you want to install:
+
+    ``` yaml title="Example CR"
+    apiVersion: clusterextension.operatorframework.io/v1alpha1
+    kind: ClusterExtension
+    metadata:
+      name: <extension_name>
+    spec:
+      packageName: <package_name>
+      channel: <channel>
+      version: "<version>"
+    ```
+
+     `extension_name`
+     : Specifies a custom name for the Kubernetes extension you want to install, such as `my-argocd`.
+
+     `package_name`
+     : Specifies the name of the package you want to install, such as `argocd-operator`.
+
+     `channel`
+     : Optional: Specifies the extension's channel, such as `stable` or `candidate`.
+
+     `version`
+     : Optional: Specifies the version or version range you want installed, such as `1.3.1` or `"<2"`.
+     If you use a comparison string to define a version range, the string must be surrounded by double quotes (`"`).
+
+    !!! warning
+        Currently, the following limitations affect the installation of extensions:
+
+        * If mulitple catalogs are added to a cluster, you cannot specify a catalog when you install an extension.
+        * OLM 1.0 requires that all of the extensions have unique bundle and package names for dependency resolution.
+
+        As a result, if two catalogs have an extension with the same name, the installation might fail or lead to an unintended outcome.
+
+2. Apply the CR the cluster:
+
+    ``` terminal
+    $ kubectl apply -f <cr_name>.yaml
+    ```
+
+    ??? success
+        ``` text title="Example output"
+        clusterextension.olm.operatorframework.io/argocd-operator created
+        ```
+
+### Verification
+
+* Get information about your bundle deployment:
+
+    ``` terminal
+    $ kubectl get bundledeployment
+    ```
+
+    ??? success
+        ``` text title="Example output"
+        NAME              ACTIVE BUNDLE   INSTALL STATE   AGE
+        argocd-operator                                   111s
+        ```
+
```

Comment:
I think this is fine to include, but in general I don't think I would expect users looking to "just get started" to go and look at whatever the underlying API we are using to apply contents to the cluster is for verification and rather just look at the `(Cluster)Extension` and read the status.

If we keep this in, could we move this below the verification of the `ClusterExtension` and add some more info that it isn't necessary to verify the `BundleDeployment` and that it is the underlying API that OLM uses to apply contents to the cluster (although maybe it is better to remove altogether since we are moving to using the Carvel `App` API)?

Reply:
I'll remove it. I added it to the downstream docs as a request from PM. But if this doesn't really add much in terms of information and it is changing, it's probably best just to cut it.